containerd-shim-v2/service.go

```diff
@@ -274,7 +298,20 @@ func getTopic(e interface{}) string {
 	return cdruntime.TaskUnknownTopic
 }
 
+func trace(ctx context.Context, name string) (opentracing.Span, context.Context) {
```

Comment:
@amshinde `Trace()` in `pkg/katautils/tracing.go` sets a tag for the cli, which we don't want in this case. I noticed virtcontainers had its own implementation of `trace()` so I implemented one for shimv2. I think we should fix this in a subsequent PR so that we're not replicating code.

Reply:
Agreed, but this seems like a simple fix, and probably easier to do in this commit than at a later time when the two functions may have diverged (they already began diverging here)

Reply:
@c3d virtcontainers has 13 different `trace()` implementations which exist to add different tags to spans (@jodh-intel, correct me if I'm wrong). I think this should be changed so that all use a general `Trace()` function that takes a list of tags, but since it is not essential to shimv2 tracing functionality and applies to many other functions, it should have its own issue & PR.

Reply:
See https://github.com/kata-containers/runtime/issues/3093